From 353b583e0ee236a8d9a520aab7b890158a7fadfe Mon Sep 17 00:00:00 2001
From: Jianing Ren <jianing.ren@rock-chips.com>
Date: Mon, 2 Mar 2020 09:53:53 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: Add HCLK_USB_PERI for rk3368
 otg"

We have removed the driver dwc3-rockchip-inno, and the hclk_usb_peri
is controlled by clock system framework instead of USB driver.

This reverts commit 24992130f2a4e2bc8ebc1d140518218dc0231307.

Change-Id: I947342389a0f85bb7f322c3481349a3ee2d56298
Signed-off-by: Jianing Ren <jianing.ren@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 1bcf18bdff76..c2deb45cf56a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -799,8 +799,8 @@
 				"snps,dwc2";
 		reg = <0x0 0xff580000 0x0 0x40000>;
 		interrupts = <GIC_SPI 23 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&cru HCLK_OTG0>, <&cru HCLK_USB_PERI>;
-		clock-names = "otg", "hclk_usb_peri";
+		clocks = <&cru HCLK_OTG0>;
+		clock-names = "otg";
 		dr_mode = "otg";
 		g-np-tx-fifo-size = <16>;
 		g-rx-fifo-size = <280>;
-- 
2.35.3

